php-classes/Emergence/Mailer/PostmarkMailer.php

```diff
@@ -50,13 +50,14 @@ static protected function apiPost($data)
 		}
 		else
 		{
-			$fh = fopen("$_SERVER[SITE_ROOT]/logs/postmark-post-fail.log", 'a');
-			fwrite($fh, date('Y-m-d h:i:s')."\t$httpStatus\t$url\tREQUEST:\$data\n--END REQUEST--\nRESPONSE:\n$result\n--END RESPONSE--\n\n");
-			fclose($fh);
-			
+            $resultDecode = json_decode($result);
```

Comment:
If the call wasn't successful we can't be sure what came back in the response. The below code will fail if json_decode returns null or something that's not an object. We should just pass the raw $result value to the logger